dcrjson/walletsvrresults.go

```diff
@@ -205,6 +205,13 @@ type ValidateAddressWalletResult struct {
 	SigsRequired int32    `json:"sigsrequired,omitempty"`
 }
 
+// VerifySeedResult models the data returned by the wallet server verify
+// seed command.
+type VerifySeedResult struct {
+	Result   string `json:"result"`
```

Comment:
why is this a string?

Reply:
What I understood from the PR is to check whether the inputted seed matches the running wallets seed.  If they do indeed match a response such as "valid seed", or "invalid seed" is returned.

Reply:
just use a bool?

Reply:
Am coming from C... Will do.

Reply:
Will do, am coming from C.